src/clusterfuzz/_internal/bot/tasks/utasks/analyze_task.py

```diff
@@ -331,15 +351,12 @@ def utask_main(uworker_input):
       uworker_input.setup_input, uworker_input.analyze_task_input.bad_revisions)
   testcase.crash_revision = environment.get_value('APP_REVISION')
 
-  analyze_task_output = uworker_io.AnalyzeTaskOutput()
-  analyze_task_output.crash_revision = int(testcase.crash_revision)
-  analyze_task_output.absolute_path = testcase.absolute_path
-  analyze_task_output.minimized_arguments = testcase.minimized_arguments
-
   if not testcase_file_path:
-    output.analyze_task_output = analyze_task_output
     return output
 
+  analyze_task_output = uworker_io.AnalyzeTaskOutput()
```

Comment:
Instead of passing a fresh output into `_update_analyze_task_output()` for it to mutate, consider returning a new object from this helper:

```py
def build_task_output(testcase) -> uworker_io.AnalyzeTaskOutput:
   ...
```

Reply:
Done.